BLSQ/iaso
# IA-2621:  Org unit change request parent shape

Add alert on whether the change request location falls within the parent's shape or not

Related JIRA tickets : IA-2621
## Self proofreading checklist



## Changes

- fixed typescript issues for react-leaflet and removed ts-ignore
- added geo_json on parent org unit in api
- hook to see if a location is within a shape
- customise markermap component to get a parent org unit and display it on map
- add popup if marker is not inside his parent shape
- covering edge case when you change parent too

## How to test

- make sure you have an org unit with a parent
- create a change request with this org unit and change the location  with something like this SRID=4326;POINT Z (23.0449 -2.9820 0)
- open change request dialog and check that everything is fine
- if you location is within the parent shape, edit the change request new location value and change lat or long to 0

## Print screen / video

![Screenshot 2024-02-01 at 12 51 05](https://github.com/BLSQ/iaso/assets/12494624/6c470b86-582a-4463-9696-f9c6cec73502)
